packages/features/form-builder/FormBuilderField.tsx

```diff
@@ -53,9 +54,20 @@ export const FormBuilderField = ({
 }) => {
   const { t } = useLocale();
   const { control, formState } = useFormContext();
+  const searchParams = useRouterQuery();
 
   const { hidden, placeholder, label } = getAndUpdateNormalizedValues(field, t);
 
+  const isPreFilledInUrl = (): boolean => {
+    const message = (formState?.errors?.responses?.message || "") as string;
+    const name = message.replace(/\{([^}]+)\}.*/, "$1");
```

Comment:
Let's abstract it out in a function
```suggestion
    const name = getFieldNameFromErrorMessage(errorMessage)
```

Also, use the function in other place as well from which this regex is copied.